lib/src/services/ds_toast.service.dart

```diff
@@ -0,0 +1,361 @@
+          padding: const EdgeInsets.only(top: 20.0, left: 20.0),
+          child: _setIcon('svg', 'assets/images/icon_success.svg'),
+        );
+        break;
+      case DSToastType.warning:
+        backgroundColor = DSColors.primaryYellowsCorn;
+        icon = Padding(
+          padding: const EdgeInsets.only(top: 20.0, left: 20.0),
+          child: _setIcon('img', 'assets/images/icon_alert_warning.png'),
+        );
+        break;
+      case DSToastType.error:
+        backgroundColor = DSColors.extendRedsFlower;
+        icon = Padding(
+          padding: const EdgeInsets.only(top: 20.0, left: 20.0),
+          child: _setIcon('img', 'assets/images/icon_alert_error.png'),
+        );
+        break;
+      case DSToastType.system:
+        backgroundColor = DSColors.illustrationBlueGenie;
+        icon = Padding(
+          padding: const EdgeInsets.only(top: 20.0, left: 20.0),
+          child: _setIcon('svg', 'assets/images/icon_blip.svg'),
+        );
+        break;
+      case DSToastType.notification:
+        backgroundColor = DSColors.neutralLightSnow;
+        icon = SizedBox(
+          width: 50.0,
+          child: ClipRRect(
+            borderRadius: BorderRadius.circular(8.0),
+            child: Image.asset(
+              'assets/images/notificacao.png',
+              package: DSUtils.packageName,
+            ),
+          ),
+        );
+        break;
+      default:
+        DSToastType.success;
+    }
+  }
+
+  /// Switches between exit button types
+  Widget _setMainButton() {
+    return actionType == DSActionType.icon
+        ? Container(
+            padding: const EdgeInsets.only(top: 10.0, right: 10.0),
+            child: IconButton(
+              onPressed: () {
+                state!(() {
+                  _stopTimer();
+                  _controlAnimation = Control.playReverse;
+                });
+              },
+              icon: const Icon(Icons.close),
+            ),
+          )
+        : Container(
+            padding: const EdgeInsets.only(top: 10.0, right: 10.0),
+            child: DSTertiaryButton(
+              onPressed: () {
+                onPressedButton!();
+                state!(
+                  () {
+                    _stopTimer();
+                    _controlAnimation = Control.playReverse;
+                  },
+                );
+              },
+              label: buttonText,
+            ),
+          );
+  }
+
+  /// Create and manage the toast animation
+  StatefulBuilder _animeCard() {
+    double inicio = (MediaQuery.of(context).size.width) * -1.0;
+    double fim = 0.0;
+
+    return StatefulBuilder(
+      builder: (BuildContext context, StateSetter mystate) {
+        state = mystate;
+        return CustomAnimationBuilder<double>(
+          duration: Duration(milliseconds: animationDuration),
+          control: _controlAnimation!,
+          tween: Tween(begin: inicio, end: fim),
+          builder: (context, value, child) {
```

Comment:
could be `(_, __, child)`

Reply:
In context it makes sense to use underscore, but in 'value' it doesn't

Reply:
Done